src/corredor/helpers/compose.ts

```diff
@@ -178,10 +261,10 @@ export class Compose {
     width = 13,
     kind = 'Record',
     options = {},
-    title = null,
+    title = '',
     style = {},
     fields,
-  }) {
+  }: PageBlockParams): PageBlock {
```

Comment:
I've talked w/ @tjerman about this.
this function is a bit too specific to be in helpers. 

Remove it please.